packages/app/src/components/Header/index.tsx

```diff
@@ -101,12 +104,18 @@ const Header = ({ data }: { data: any }) => {
    * This function saves the 'data', 'rows', 'visibleSets', 'hiddenSets', and query parameters to the local storage.
    * TODO: resolve cache size limit issues for large datasets (10mb)
    */
-  const dispatchState = () => {
-    localStorage.setItem('data', JSON.stringify(data));
-    localStorage.setItem('rows', JSON.stringify(getAccessibleData(getRows(data, provenance.getState()), true)));
-    localStorage.setItem('visibleSets', JSON.stringify(visibleSets));
-    localStorage.setItem('hiddenSets', JSON.stringify(hiddenSets.map((set: Column) => set.name)));
+  const dispatchState = async () => {
```

Comment:
Is there any issue using `function` notation here? I think that's more readable generally than assigning a lambda to a variable

Reply:
I agree, I much prefer `function` syntax